models/mail.go

```diff
@@ -148,6 +148,24 @@ func composeTplData(subject, body, link string) map[string]interface{} {
 	return data
 }
 
+func composeIssueCommentMessage(issue *Issue, doer *User, comment *Comment, tplName base.TplName, tos []string, info string) *mailer.Message {
+	subject := issue.mailSubject()
+	body := string(markdown.RenderString(issue.Content, issue.Repo.HTMLURL(), issue.Repo.ComposeMetas()))
+	data := composeTplData(subject, body, issue.HTMLURL() + "#" +  comment.HashTag())
```

Comment:
There's `comment.HTMLURL()` to replace `issue.HTMLURL() + "#" + comment.HashTag()` ... Also `go-fmt` :)